apps/prairielearn/src/components/PageLayout.tsx

```diff
@@ -92,11 +92,11 @@ export function PageLayout({
 
     const sideNavExpanded = sideNavEnabled && resLocals.side_nav_expanded;
 
-    let showContextNavigation = true;
+    let showContextNavigation = sideNavEnabled;
```

Comment:
This makes the tabs invisible on the global admin pages and the institution admin pages.